src/documents/tests/test_api.py

```diff
@@ -3532,6 +3532,44 @@ def test_object_permissions(self):
             status.HTTP_404_NOT_FOUND,
         )
 
+    def test_api_set_permissions(self):
+        user1 = User.objects.create_superuser(username="user1")
+        user2 = User.objects.create(username="user2")
+
+        self.client.force_authenticate(user1)
+
+        response = self.client.post(
+            "/api/tags/",
+            json.dumps(
+                {
+                    "name": "test1",
+                    "matching_algorithm": MatchingModel.MATCH_AUTO,
+                    "set_permissions": {
+                        "owner": None,
```

Comment:
Does a none owner with other permissions make sense?  Wouldn't a none owner be open to all anyway?

Reply:
Ah, good point. I was trying to be compact but I'll split it up.